weave/environment.py

```diff
@@ -136,14 +136,19 @@ def analytics_disabled() -> bool:
 
 
 def weave_server_url() -> str:
-    base_url = wandb_base_url()
+    base_url = wandb_frontend_base_url()
     default = "https://weave.wandb.ai"
     if base_url != "https://api.wandb.ai":
         default = base_url + "/weave"
     return os.getenv("WEAVE_SERVER_URL", default)
 
 
 def wandb_base_url() -> str:
+    pod_url = os.getenv("WANDB_POD_BASE_URL", "").rstrip("/")
```

Comment:
nit: i'd prefer if this were named something like `WANDB_PRIVATE_BASE_URL` -- pod is ambiguous here, and a misnomer in some cases